src/tests/bulkConcepts/containers/BulkConceptsPage.test.js

```diff
@@ -297,7 +297,58 @@ describe('Test suite for BulkConceptsPage component', () => {
     wrapper.find('#add-button').simulate('click');
     jest.runAllTimers();
   });
-
+  it('should check whether the concept referenced in recursiveQuestionAnswers has a mapping', () => {
```

Comment:
Is it the test that should check whether recursiveQuestionAnswers has a mapping? Or is the test ensuring that the tested code does so?

Reply:
The test ensures that the tested code has a mapping. let me rephrase the description accordingly

Reply:
Done, is the new description satisfactory?

Reply:
Can you share this with your team mates?

Reply:
I have had a discussion with the teammates and adjusted the description accordingly, does it suffice?

Reply:
I have had a discussion with the teammates and adjusted the description accordingly, does it suffice?